extra_data/stacking.py

```diff
@@ -131,6 +132,59 @@ def stack_detector_data(train, data, axis=-3, modules=16, fillvalue=None,
     return stack
 
 
+def stack_from_xarray(xrdata, axis=-3, modules=16, fillvalue=None,
+                      real_array=True):
+    """Stack data from a xarray.
+
+    Parameters
+    ----------
+    xrdata: xarray
+        (Big) detector data. Expected dims are:
+        ('module', 'train', 'pulse', 'slow_scan', 'fast_scan')
```

Comment:
I think this only expects that there is a `module` dimension, right? The other names shouldn't matter.